src/main/java/hudson/plugins/jira/JiraJobAction.java

```diff
@@ -31,7 +32,7 @@
  * e.g. "JENKINS-1234" and "feature/JENKINS-1234" will have this action with the issue JENKINS-1234 referenced
  */
 @ExportedBean
-public class JiraJobAction implements Action {
+public class JiraJobAction implements Action, Serializable {
```

Comment:
Why is this required

Reply:
good catch sorry I miss to remove this. done

Reply:
Please organise imports - https://sonarcloud.io/project/issues?id=jenkins-jira-plugin&open=AXIIy9Pg99-5isQx0vpU&pullRequest=239&resolved=false&types=CODE_SMELL